src/index.ts

```diff
@@ -79,7 +79,7 @@ export const beautifier: Beautifier = {
           }
         }
       ],
-      ternaryline: "preserve_ternary_lines"
+      ternaryline: "multiline_ternary"
```

Comment:
I am fairly certain this is a bug.

`multiline_ternary` can be `always`, `always-multiline`, or `never`. 
I think `ternaryline` expects `boolean`. So will need a transform option handler.